### PR TITLE
Refactoring before final submission

### PR DESCRIPTION
**Pull request by dasGoogle**
This PR touches some details of the configuration, updates aspects where there have been changes in the code that make the documentation obsolete. 

It also attempts to unify the language used in the documentation.

### CODE REVIEW

File: docs/setup/ansible_setup.md [hunk 1]
```diff
@@ -10,33 +10,33 @@ This guide will show you how you can install you own WiMoVE setup using our ansi
 
 The parts of this guide will provide information on how to set up each component. Before getting started, please make sure you understand the purpose of each component in the whole system by reading the [Architecture page](../../architecture/).
 
-WiMoVE provides a package you can install on your OpenWrt router but needs some additional services in your network to be useful:
+WiMoVE provides a package you can install on your OpenWrt router (WiMoVEd) but needs some additional services in your network to be useful:
```
Comment (rgwohlbold) [inline docs/setup/ansible_setup.md]: We should use `wimoved` here

File: docs/setup/ansible_setup.md [hunk 2]
```diff
@@ -71,51 +71,52 @@ By default, we assume that the same interface is used for both internet access a
...
 !!! tip
 
     You can use arbitrary model names. The `directory` variable for a host only has to match the directory name in `roles/access-points/files` to get the correct binary.
 
 ## Generate OpenWrt configuration files
 
-Since each AP is different, we need different kinds of configuration files.
-For this reason, we do not provide configuration files for your APs but rather guide you how to create them yourself.
+Since each AP is different, you need different configuration files for each model.
+For this reason, we do not provide complete configuration files APs but rather guide you how to create them yourself.
```
Comment (rgwohlbold) [inline docs/setup/ansible_setup.md]: There seems to be one `APs` to much here

File: docs/setup/ansible_setup.md [hunk 3]
```diff
@@ -71,51 +71,52 @@ By default, we assume that the same interface is used for both internet access a
...
 
-1. Go to the web interface of one of your APs and configure a WPA2-PSK wireless network for each radio you wish to use. You can set any SSID and password, we will overwrite them in a minute. Make sure to press save at the end.
-1. Open an SSH console to the AP and run
+1. Go to the web interface of one of your APs running OpenWRT and configure a WPA2-PSK wireless network for each radio you wish to use. You can set any SSID and password, you will overwrite them in a minute. Make sure to press save at the end.
```
Comment (rgwohlbold) [inline docs/setup/ansible_setup.md]: We should use the spelling `OpenWrt`

File: docs/setup/manual/access_point.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
 # Setting Up an Access Point
 
-This part of the guide will show you how to set up wimoved on your OpenWrt Access Point.
+This part of the guide will show you how to set up WiMoVEd on your OpenWrt Access Point.
```
Comment (rgwohlbold) [inline docs/setup/manual/access_point.md]: We should consistently use the spelling `wimoved`

File: docs/setup/manual/index.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ This guide will show you how you can set up your own WiMoVE environment manually
 
 !!! tip
 
-    Since we often update our software, some details about the setup instructions might be out-of-date. In case you run into problems, feel free to take a look at the [Ansible deployment scripts](https://github.com/WiMoVE-OSS/deployment) or [Create a new Issue](https://github.com/WiMoVE-OSS/docs/issues/new).
+    Since we often modify our software, some details about the setup instructions might be out-of-date. In case you run into problems, feel free to take a look at the [Ansible deployment scripts](https://github.com/WiMoVE-OSS/deployment) or [Create a new Issue](https://github.com/WiMoVE-OSS/docs/issues/new).
```
Comment (rgwohlbold) [inline docs/setup/manual/index.md]: I would leave out the "might be out of date". It sounds unprofessional if the documentation warns the reader about being out-of-date

File: docs/setup/manual/parameters.md [hunk 6]
```diff
@@ -2,47 +2,49 @@
...
+## Gateway IP Address
 
 This is the IP address where the gateway can be reached.
 
 ## Route Reflector IP Address
 
 This is the IP address at which the route reflector can be reached for both the APs and the gateway.
 
-The route reflector functionality can also be included in the gateway. This is however currently only described in the ansible deployment guide.
+The functionality of the route reflector can also be included in the gateway. This is however currently only described in the Snsible deployment guide. You will need to combine the FRR configuration files for the route reflector and gateway to get this to work.
```
Comment (rgwohlbold) [inline docs/setup/manual/parameters.md]: Ansible :)

File: docs/setup/manual/route_reflector.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
-# Setting Up a Route Reflector
+# Setting Up a route reflector
```
Comment (rgwohlbold) [inline docs/setup/manual/route_reflector.md]: We should use consistent casing in all documentation titles. I would prefer Title Case, but that needs to be used everywhere

**Review by rgwohlbold — CHANGES_REQUESTED** [review]
Some small issues

File: docs/setup/manual/index.md (hunk 5 above)
Comment (dasGoogle) [reply docs/setup/manual/index.md]: See if the new version fits better

**Review by rgwohlbold — APPROVED** [review]
:shipit:

**Review by Sohn123 — APPROVED** [review]
Nice!

**Pull request merged**